packages/publisher/bitbucket/src/PublisherBitbucket.ts

```diff
@@ -0,0 +1,78 @@
+import PublisherBase, { PublisherOptions } from '@electron-forge/publisher-base';
+import { asyncOra } from '@electron-forge/async-ora';
+
+import fetch from 'node-fetch';
+import FormData from 'form-data';
+import fs from 'fs-extra';
+import path from 'path';
+
+import { PublisherBitbucketConfig } from './Config';
+
+export default class PublisherGithub extends PublisherBase<PublisherBitbucketConfig> {
+  name = 'bitbucket';
+
+  async publish({ makeResults }: PublisherOptions) {
+    const { config } = this;
+    const hasRepositoryConfig = config.repository && typeof config.repository;
+    const overrideExistingVersion = Boolean(config.overrideExistingVersion);
+    const appPassword = process.env.BITBUCKET_APP_PASSWORD;
+    const username = process.env.BITBUCKET_USERNAME;
+    const auth = Object.assign({}, { appPassword, username }, config.auth || {});
+    // TODO: See if we can use this same API for bitbucket server, we could take in a `host` config if so
+    const apiUrl = `https://api.bitbucket.org/2.0/repositories/${config.repository.owner}/${config.repository.name}/downloads`;
+    const encodedUserAndPass = Buffer.from(`${auth.username}:${auth.appPassword}`).toString('base64');
+
+    if (!(hasRepositoryConfig && config.repository.owner && config.repository.name)) {
+      throw 'In order to publish to Bitbucket you must set the "repository.owner" and "repository.name" properties in your forge config. See the docs for more info'; // eslint-disable-line
```

Comment:
I generally prefer to specify which eslint rule I'm ignoring here.

Reply:
I honestly couldn't tell you which it was, I copied it from the GitHub one, probably max-len. 

I'll update both tomorrow.